common/changes/@itwin/property-grid-react/leck-property-grid_2022-06-22-19-35.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/property-grid-react",
+      "comment": "Replace deprecated Table with MenuItem from iTwinUI-React",
+      "type": "patch"
```

Comment:
i would make this a minor bump, just because its a new component and new dep
you should be able to delete the changelog entry, common/changes/@itwin/property-grid-react/leck-property-grid_2022-06-16-20-01.json, and keep this one. but make sure to mention in this log that this fixes the findDomNode warning